README.md

```diff
@@ -18,12 +18,37 @@ This code enables tunneling of a single threaded TCP client / server socket inte
 
 * Docker 18+
 
-### Running the Docker Build
+### Using Pre-built Docker Images
 
-`./docker-build.sh`
+We provide several docker images on various platforms.
+There are two types of images: base images and release images.
+The base images come with all dependencies pre-installed. You will still need to download and build the source.
+These are useful if you want to modify and [compile](https://github.com/aws-samples/aws-iot-securetunneling-localproxy#download-and-build-the-local-proxy) the local proxy on your own, but are large (~1 GB each).
+You can find them at:
+#### https://gallery.ecr.aws/aws-iot-securetunneling-localproxy/base-images
+The release images are minimum size images that include a pre-built binary. No dependencies are installed, so recompilation will not work.
+The format of every tag is `[arch]-[os]-[git commit sha]` for example: amd64-ubuntu-33879dd7f1500f7b3e56e48ce8b002cd9b0f9e4e.
```

Comment:
I actually did this poorly in aws-iot-device-client so I would suggest changing the way this all works.

You don't need to specify the architecture I think... when we build & push we the GitHub action using the different architectures it should just work like that out of the box. Then you can get pull for a specific architecture.

I would try something like creating ECR repositories like /aws-iot-local-proxy/ubuntu and /aws-iot-local-proxy/amazonlinux and then in each repository have the tag be 'aws-iot-local-proxy-[git commit sha] instead. The way we have it for device client is confusing.